fastchat/conversation.py

```diff
@@ -341,6 +370,15 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# An empty template for raw conversation with roles named, used for loading huggingface chat templates
+register_conv_template(
+    Conversation(
+        name="base",
+        roles=("User", "Assistant"),
```

Comment:
Not sure if there is a cleaner way to do this, but this is basically the same as the raw one, but I added roles so they show up in the cli. Basically, any model that uses the huggingface template will hit this conversation template